tests/serialization/test_arrow_compat.py

```diff
@@ -49,6 +49,8 @@ def test_arrow_compat(arrow_version, reference_store, mocker):
     restored = ParquetSerializer().restore_dataframe(
         store=reference_store, key=arrow_version + ".parquet", date_as_object=True
     )
-    if arrow_version == "0.14.1" and not ARROW_LARGER_EQ_0141:
+
+    if arrow_version in ("0.14.1", "0.15.0", "0.16.0") and not ARROW_LARGER_EQ_0141:
```

Comment:
Wouldn't the `ARROW_LARGER_EQ_0141` be sufficient?

Reply:
No, under pyarrow 0.13.1 (lib) the fix is not required for the 0.12 and 0.13 reference files (`arrow_version` => stored reference file, `ARROW_LARGER_EQ_0141` => current lib/runtime version)